packages/deamon/src/main.rs

```diff
@@ -0,0 +1,86 @@
+#[macro_use]
+extern crate lazy_static;
+
+use anyhow::Result;
+use config::{process_json_files_in_folder, NydusConfig};
+use homedir::get_my_home;
+use pid::{check_projects, init_projects};
+use server::start_server;
+use tokio::{select, sync::mpsc};
+use utils::create_folder_if_not_exists;
+
+mod config;
+mod pid;
+mod server;
+mod utils;
+
+fn setup_logger() -> Result<()> {
+    let log_dir = get_my_home()
+        .unwrap()
```

Comment:
We should avoid using 'unwrap'.

Reply:
this function is init, it should panic when it error

Reply:
It's better to use 'Error' for exiting or panicking at a higher level. We should consider the potential scenarios where this method might be used in the future.

Reply:
done